frontend/include/chpl/util/version-info.h

```diff
@@ -71,9 +71,9 @@ bool getIsOfficialRelease();
 
```

Comment:
For the comment for  `getIsOfficialRelease`: would you mind capitalizing `Get` above and having the comment mention that the CMakeLists.txt variable is `CHPL_OFFICIAL_RELEASE` ?